### ISSUE TITLE
Is usage of SQL Server Native Client in the data source supported?

### ISSUE THREAD

**Issue by Utwig1** [otykier/TabularEditor#295]
Hello,

I've initially created my tabular project in SSDT and set it up to use the SNAC provider (SQLNCLI11). I know that SNAC was declared deprecated some time ago, and I plan to move to currently supported OLEDB provider - MSOLEDB. OLEDB is my way to go because it provides much better performance than .NET provider.
Unfortunately I can't select either SNAC or MSOLEDB when working from the Tabular Editor 2.8.2 (refreshing metadata, adding new tables etc.), only .NET provider.
Do I miss something or is it just an open issue that only .NET providers are supported from Tabular Editor at the moment?

Thanks!

Andrej Kuklin

**Comment by otykier** [comment]
Maybe I'm misunderstanding something, but would the below options not be equivalent to SNAC?

![image](https://user-images.githubusercontent.com/8976200/59470260-76dc3800-8e37-11e9-9361-25880ccac48b.png)

---

Also, would the below options not be equivalent to OLEDB?

![image](https://user-images.githubusercontent.com/8976200/59470366-bc006a00-8e37-11e9-9d26-38532a51d5ee.png)

**Comment by Utwig1** [comment]
The first screenshot shows .NET Framework Provider (the description on the right side says this explicitly), not SNAC.

I've found an acceptable workaround in the meantime. I'm connecting to my datasource using a SQL login, not Integrated Security. The password for SQL login is not stored in the connection string, so not surprisingly the attempt to refresh table metadata fails. To fix this I clicked on "..." right to the connection string, which brought me the connection properties dialog. This was NOT the best way to solve the problem, because I can only select .NET Framework Providers in this dialog.
What I'm doing now instead is just editing the connection string manually (adding Password=123;). I can also save the bim file, the password is saved as "***" and needs to be reentered if I close and open the Tabular Editor (which is fine).
I'm closing the issue, thank you for this great tool :)
